js/fragment_shader.js

```diff
@@ -191,8 +191,26 @@ bool intersectSomething(vec3 origin, vec3 rayDirection, out vec3 intersection,
   return true;
 }
 
+bool intersectSomethingGoingToLight(vec3 origin, vec3 N_aux) {
```

Comment:
This function checks if the V ray can reach the light without intersecting any object on the way.